### DISCUSSION TITLE
Update PlugRing records plz

### DISCUSSION THREAD

**Discussion by d4rkmen** [FarGroup/FarManager#839] [General]
The dev account is suspended.
Just want to update plugin version

https://github.com/d4rkmen/PhoneFS

Thanks in advance

**Comment by johnd0e** [comment]
Plugring is broken..
There is just no "Darkmen" user at all.
And it's impossible to edit "PhoneFS" record, even for admin.

**Comment by johnd0e** [comment]
@d4rkmen 
Fixed https://plugring.farmanager.com/plugin.php?pid=866&l=ru
